crates/holochain/src/conductor/tests/cell_cloning.rs

```diff
@@ -139,6 +140,81 @@ async fn create_clone_cell_run_twice_returns_correct_clones() {
     assert_eq!(clone_cell_1.original_dna_hash, dna.dna_hash().to_owned());
 }
 
+#[tokio::test(flavor = "multi_thread")]
+async fn create_identical_clone_cell_twice_fails() {
+    let (dna, _, _) = SweetDnaFile::unique_from_test_wasms(vec![TestWasm::Create]).await;
+    let role_name: RoleName = "dna_1".to_string();
+    let mut conductor = SweetConductor::from_standard_config().await;
+    let (alice, bob) = SweetAgents::two(conductor.keystore()).await;
+    let apps = conductor
+        .setup_app_for_agents(
+            "app",
+            &[alice.clone(), bob.clone()],
+            [&(role_name.clone(), dna.clone())],
+        )
+        .await
+        .unwrap()
+        .into_inner();
+    let alice_app = &apps[0];
```

Comment:
There may be a more elegant way to do this. `into_tuple` didn't work out for me.